application/src/test/java/org/thingsboard/server/controller/BaseDashboardControllerTest.java

```diff
@@ -73,18 +79,18 @@ public void beforeTest() throws Exception {
     public void afterTest() throws Exception {
         loginSysAdmin();
 
+        afterTestEntityDaoRemoveByIdWithException (dashboardDao);
+
         doDelete("/api/tenant/" + savedTenant.getId().getId().toString())
                 .andExpect(status().isOk());
     }
 
     @Test
     public void testSaveDashboard() throws Exception {
-        Dashboard dashboard = new Dashboard();
```

Comment:
refactoring is not welcomed in the bug-fix PRs. Please, maintain as few as possible changes. You can do a major refactor in the next PR

Reply:
OK